readthedocs/rtd_tests/tests/test_footer.py

```diff
@@ -448,7 +448,7 @@ def test_highest_version_without_tags(self):
 class TestFooterPerformance(TestCase):
     # The expected number of queries for generating the footer
     # This shouldn't increase unless we modify the footer API
-    EXPECTED_QUERIES = 12
+    EXPECTED_QUERIES = 11
```

Comment:
Caching the  `get_parent_relationship` method reduced one query here :)

Reply:
I'd like to have more tests like this for El Proxito itself. I know you were reducing the amount of db queries there as well and it's awesome to see these numbers going down 💯